boards/veer_el2_sim/README.md

```diff
@@ -0,0 +1,68 @@
+# VeeR EL2 simulation target
+
+For more information about the core and the Verilator testbench [visit the VeeR EL2 repository](https://github.com/chipsalliance/Cores-VeeR-EL2).
+This was tested with Verilator 5.006, which can be installed as explained [in the doc](https://verilator.org/guide/latest/install.html).
+
+There's also a repository dedicated to [running Tock on VeeR EL2 in simulation](https://github.com/chipsalliance/VeeR-EL2-tock-example)
+
+## Building software
+
+### Building Tock
+
+In order to compile Tock and convert it to the format used by Verilator, run:
+
+    make -C boards/veer_el2_sim debug -j$(nproc)
+    riscv64-unknown-elf-objcopy -O verilog target/riscv32imc-unknown-none-elf/debug/veer_el2_sim.elf kernel.hex
```

Comment:
This should be run by the Makefile

Reply:
Alright, I've added a proper target to the Makefile and a comment to explain why this is needed.